spec/presenters/user_navigation_spec.rb

```diff
@@ -3,54 +3,63 @@
 require 'rails_helper'
 
 describe UserNavigation do
-  let(:artist) { create :artist }
-  let(:artist_nav) { described_class.new(artist) }
-
   describe 'remind_for_open_studios_register?' do
-    let(:artist_to_remind) { create(:artist) }
-    let(:artist_to_remind_nav) { described_class.new(:artist_to_remind) }
```

Comment:
I think this may have made us miss the fails.  This should have been `described_class.new(artist_to_remind)` but instead the "artist" was just the symbol `:artist_to_remind`.  It'd kind of funny that these tests didn't catch that.

anyway, I reworked the test suite so I could add `when user is not logged in` and `when user is logged in but not as an Artist`.